ci/tests.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
   - python-casacore>=3.1.0
   - pytest
   - pytest-cov
-  - pyuvdata>=2.2.8
+  - pyuvdata>=2.2.10
```

Comment:
there's one more CI yaml, `publish.yaml` that also needs to be updated with this change.